contracts/BFactory.sol

```diff
@@ -36,21 +41,34 @@ contract BFactory is BBronze {
         return _isBPool[b];
     }
 
+    function make_payable(address x) internal pure returns (address payable) {
+        return address(uint160(x));
+    }
+
     function newBPool()
         external
         returns (BPool)
     {
-        BPool bpool = new BPool();
-        _isBPool[address(bpool)] = true;
-        emit LOG_NEW_POOL(msg.sender, address(bpool));
-        bpool.setController(msg.sender);
-        return bpool;
+        bytes memory bytecode = type(InitializableProxy).creationCode;
```

Comment:
why not use `Clone` from openZeppelin? What is the size of `InitializableProxy`?
https://github.com/CoverProtocol/cover-core-v2/blob/main/contracts/proxy/Clones.sol#L24

Reply:
Will use, thank you